docs/extensions/api.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Extension API
+category: TODO
+hidden: true
+---
+
+Since CLI extensions are built on top of the Deno runtime, they have access to
+two different APIs; the Deno API and the Phylum API.
+
+## Deno API
+
+Deno's API is built into the Deno runtime, providing access to all external
+interfaces like Network, Disk, or the terminal console. All available
+functionality is documented in [Deno's API docs]. This functionality is
+available to all extensions without any imports.
+
+Additionally, Deno also provides a complementary standard library. This includes
+utility functions for several commonly used structures like collections, http,
+and async. These modules can be imported using the URLs documented in Deno's STD
+documentation or by downloading them and including individual modules as files.
+All STD functionality is documented in [Deno's STD docs]
```

Comment:
```suggestion
All STD functionality is documented in [Deno's STD docs].
```

In general I'd use an explicit "Deno's standard library" unless the uppercase STD has a special meaning.